app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -1,3 +1,4 @@
+- display_grading_results = @submission.published? || can?(:grade, @submission)
```

Comment:
Any reason why ?  We don't use it elsewhere. This line is for caching the result, so we don't call `can?` multiple times.